apps/accelerate/open_alpha_tensor/open_alpha_tensor/core/actors/stage.py

```diff
@@ -125,19 +150,15 @@ def select_future_state(
     possible_states: List[torch.Tensor],
     q_values: torch.Tensor,
     N_s_a: torch.Tensor,
-    repetitions: Dict[int, list],
+    repetitions: Dict[int, int],
```

Comment:
please keep the old code modification (the code in red should replace the code in green)